public/asset/admin/js/sb-admin.js

```diff
@@ -0,0 +1,18 @@
+$(function() {
+
+    $('#side-menu').metisMenu();
+
+});
+
+//Loads the correct sidebar on window load,
+//collapses the sidebar on window resize.
+$(function() {
+    $(window).bind("load resize", function() {
+        width = (this.window.innerWidth > 0) ? this.window.innerWidth : this.screen.width;
+        if (width < 768) {
```

Comment:
can we put those 768 number into constant ?

Reply:
I created file constant for this case